docs/docs/reconciliation.md

```diff
@@ -73,21 +67,19 @@ renderB: <div style={{'{{'}}fontWeight: 'bold'}} />
 
 After the attributes have been updated, we recurse on all the children.
 
-
 ### Custom Components
 
 We decided that the two custom components are the same. Since components are stateful, we cannot just use the new component and call it a day. React takes all the attributes from the new component and calls `componentWillReceiveProps()` and `componentWillUpdate()` on the previous one.
```

Comment:
The first sentence in this paragraph is confusing. When did we decide that? I would try to merge some of the sentences about custom components in "Different Node Types" with this section, maybe by moving them and the associated example here.